src/Places/Suggestions/SuggestionsType.ts

```diff
@@ -0,0 +1,60 @@
+import { Airport, City } from '../../types'
+
+export interface PlacesSuggestionsResponse {
+  /**
+   * The type of the place
+   */
+  type: string
+  /**
+   * The time zone of the airport, specified by name from the tz database
+   */
+  time_zone: string | null
+  /**
+   * The name of the place
+   */
+  name: string
+  /**
+   * The longitude position of the airport represented in Decimal degrees with 6 decimal points with a range between -180° and 180°
+   */
+  longitude: number | null
```

Comment:
Just making sure this is a string not a number?

Reply:
Yeah, it's a number!

<img width="741" alt="image" src="https://user-images.githubusercontent.com/2954009/179247836-a12c1af7-305f-4f1e-8e52-3a50e3603805.png">

Reply:
Nice, thanks for checking!